Task3/task1/tableController.js

```diff
@@ -0,0 +1,72 @@
+﻿const downloads = [
+  {
+    id: 1,
+    title: "Recipe",
+    status: "Done",
+  },
+  {
+    id: 2,
+    title: "Workouts",
+    status: "Pending",
+  },
+  {
+    id: 3,
+    title: "Passwords",
+    status: "Pending",
+  },
+  {
+    id: 4,
+    title: "To Do 2021",
+    status: "Pending",
+  },
+  {
+    id: 5,
+    title: "Books",
+    status: "Failed",
+  },
+];
+
+function fillTableByInitialValues() {
+  const idRow = document.getElementById("idRow");
+  const titleRow = document.getElementById("titleRow");
+  const statusRow = document.getElementById("statusRow");
+
+  for (let i = 0; i < downloads.length; i++) {
+    idRow.innerHTML += `<td>${downloads[i].id}</td>`;
+    titleRow.innerHTML += `<td>${downloads[i].title}</td>`;
+    statusRow.innerHTML += `<td>${downloads[i].status}</td>`;
+  }
+}
+
+window.onload = () => fillTableByInitialValues();
+
+function findCellByTextContent(row, contentString) {
+  console.log("Check started");
+  for (let i = 0; i < row.cells.length; i++) {
+    if (row.cells.item(i).innerText === contentString) {
+      console.log("Found pending item");
+      return row.cells.item(i);
+    }
+  }
+  console.log("No more pending items left");
+  return null;
+}
+
+function updateStatusRow() {
+  const statusRow = document.getElementById("statusRow");
+
+  setTimeout(() => {
+    const findPendingCell = () => findCellByTextContent(statusRow, "Pending");
```

Comment:
Maybe it will be better to create an additional constant with all statuses? It will help us to avoid any typos in the future.
```suggestion
    const DOWNLOAD_STATUSES = {
      pending: "Pending",
      done: "Done",
      failed: "Failed"
    }
    const findPendingCell = () => findCellByTextContent(statusRow, DOWNLOAD_STATUSES.pending);
```